Update document-picker.md (#18514)
expo/expo: docs/pages/versions/unversioned/sdk/document-picker.md

- Set the `usesIcloudStorage` key to `true` in your **app.json** as specified [configuration properties](/versions/latest/config/app/#usesicloudstorage).
- You need to enable the iCloud Application Service in your App identifier. This can be done in the detail of your [App ID in the Apple Developer interface](https://developer.apple.com/account/ios/identifier/bundle).
- Enable iCloud service with CloudKit support, and create an iCloud Container. When registering the new Container, you are asked to provide a description and identifier for the container. You may enter any name under the description. Under the identifier, add `iCloud.<your_bundle_identifier>`.

> - Define a plugin in your `app.json` file to define a `iCloudContainerEnvironment` entitlement for your app:

Reviewer: - Add `expo-document-picker` [config plugin](/guides/config-plugins/) in **app.json** or **app.config.js** and pass `iCloudContainerEnvironment` entitlement as an option to the plugin:

> - Define a plugin in your `app.json` file to define a `iCloudContainerEnvironment` entitlement for your app:

Reviewer: ```suggestion
- Add `expo-document-picker` [config plugin](/guides/config-plugins/) in **app.json** or **app.config.js** and pass `iCloudContainerEnvironment` entitlement as an option to the plugin:
```

```

>     "plugins": [
>         [
>             "expo-document-picker",
>                 {
>                     "iCloudContainerEnvironment": "Production"
>                 }
>         ]
>     ],
> 

Reviewer: ```suggestion
{
  "plugins": [
    [
      "expo-document-picker",
      {
        "iCloudContainerEnvironment": "Development | Production"
      }
    ]
  ]
}
```

> ```
>     "plugins": [
>         [
>             "expo-document-picker",
>                 {
>                     "iCloudContainerEnvironment": "Production"
>                 }
>         ]
>     ],
> 
> ```

Reviewer: ```suggestion
```json
{
  "plugins": [
    [
      "expo-document-picker",
      {
        "iCloudContainerEnvironment": "Development | Production"
      }
    ]
  ]
}
```
```

The code block doesn't format as per the docs guidelines.

> 

Reviewer: ```suggestion

In the above snippet, `iCloudContainerEnvironment` takes either `Development` or `Production` as the value depending on which environment you create the iCloud container.

```
